### DISCUSSION TITLE
Line color change in the middle

### DISCUSSION THREAD

**Discussion by francisxrousseau** [recharts/recharts#5289] [Q&A]
I’d like to achieve the following on the line chart:

I have a red dot at a specific point on the line.
From this red dot, I need the line color to change for the next +2 hours on the x-axis.
After the +2 hours, the line should return to its original color.
How can I implement this behavior?

**Comment by ckifer** [comment]
You can use linear gradients to change the color from the mouse to the end of the line (there's a few examples around but I would need to dig for them). But you won't really be able to change it just from one datapoint to another since it's all one combined line. 

You could maybe fake it by creating lines with different colors that are rendered on top of the existing line but not sure how well that would work
